incubator/kured/templates/daemonset.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: extensions/v1beta1
```

Comment:
should this be apps/v1? https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

Reply:
Agreed. Changed and tested, still works.